testing/lxterminal-gtk3/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+# Contributor: Daniel YC Lin <dlin.tw@gmail.com>
+# Maintainer: Daniel YC Lin <dlin.tw@gmail.com>
+pkgname=lxterminal-gtk3
+_pkgname=lxterminal
+pkgver=0.3.0
+pkgrel=0
+pkgdesc="Desktop-independent VTE-based terminal emulator support gtk3 IME input"
+url="http://lxde.org/"
+arch="all"
+license="GPL2+"
+depends="vte3 gtk+3.0"
```

Comment:
These dependencies are automatically detected via dynamic links, so it’s not needed (nor desirable) to specify it here.

```
>>> lxterminal-gtk3*: Tracing dependencies...
	vte3
	gtk+3.0
	so:libc.musl-x86_64.so.1
	so:libgdk-3.so.0
	so:libgdk_pixbuf-2.0.so.0
	so:libglib-2.0.so.0
	so:libgobject-2.0.so.0
-->	so:libgtk-3.so.0
	so:libintl.so.8
	so:libpango-1.0.so.0
-->	so:libvte-2.91.so.0
```